### PR TITLE
Remove warnings

### PR DESCRIPTION
**Pull request by danilodileo**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/metatdenovo/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/metatdenovo _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: .github/workflows/ci.yml
Comment (erikrikarddaniel) [inline .github/workflows/ci.yml:1]: I'm not sure how this works, so I can't comment on the part from `profiles:`. If you know, fine!

**Review by erikrikarddaniel — APPROVED** [review]
Looks good, with a comment regarding my ignorance.

File: .github/workflows/ci.yml
Comment (danilodileo) [reply .github/workflows/ci.yml:1]: I followed CI.yml from Ampliseq but unfortunately is not working so there is something more to do.

Comment (erikrikarddaniel) [reply .github/workflows/ci.yml:1]: OK, I'll merge as it is.

**Pull request merged**